tests/Aspire.Dashboard.Tests/ResourceOutgoingPeerResolverTests.cs

```diff
@@ -25,6 +25,7 @@ private static ResourceViewModel CreateResource(string name, string? serviceAddr
             Properties = FrozenDictionary<string, Value>.Empty,
             Urls = servicePort is null || servicePort is null ? [] : [new UrlViewModel(name, new($"http://{serviceAddress}:{servicePort}"), isInternal: false)],
             State = null,
```

Comment:
Should we rename State to StateName?

Reply:
I have no strong opinion on this, but I'm happy to make the change if you would like - I will merge now but please respond if so. The attribute name from gRPC is state